Jenkinsfile

```diff
@@ -19,11 +18,25 @@ pipeline {
         stage('Fedora unit') {
           steps {
             sh readFile(file: 'jenkins/python3-tests.sh')
-            junit('coverage.xml')
-            // TODO: find the correct adapter or generate coverage tests that can be
-            //       parsed by an existing adapter:
-            //       https://plugins.jenkins.io/code-coverage-api/
-            // publishCoverage adapters: [jacocoAdapter('coverage.xml')]
+          }
+          post {
+            always {
+              archiveArtifacts allowEmptyArchive: true, artifacts: 'coverage.xml', fingerprint: true
+              // https://www.jenkins.io/doc/pipeline/steps/cobertura/
+              step([$class: 'CoberturaPublisher',
+                            autoUpdateHealth: false,
+                            autoUpdateStability: false,
+                            coberturaReportFile: 'coverage.xml',
+                            failNoReports: false,
+                            failUnhealthy: false,
+                            failUnstable: false,
+                            maxNumberOfBuilds: 10,
+                            onlyStable: false,
+                            sourceEncoding: 'ASCII',
+                            zoomCoverageChart: false])
+              // https://www.jenkins.io/doc/pipeline/steps/junit/
+              junit allowEmptyResults: true, keepLongStdio: true, skipMarkingBuildUnstable: true, skipPublishingChecks: true, testResults: 'coverage.xml'
```

Comment:
I'm still trying to figure out why the junit decided to stop working, but I added some flags that make this non blocking.  If it turns out to be a plugin/jenkins problem we won't have to edit this line to get the junit tests section working again.